### PR TITLE
Fix URL links on README.rst

### PR DESCRIPTION
**Pull request by marcelotrevisani**

### CODE REVIEW

**Review by Mariatta — APPROVED** [review]
Thank you. I checked the links and confirmed that the new urls are more up to date.

**Comment by Mariatta** [comment]
Ah but the CI failed somehow. Please check the errors and ensure builds are passing.

**Review by Mariatta — CHANGES_REQUESTED** [review]
You need to update the url in line 35 too.

https://github.com/brettcannon/gidgethub/pull/135/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR35

**Comment by marcelotrevisani** [comment]
> You need to update the url in line 35 too.
> 
> https://github.com/brettcannon/gidgethub/pull/135/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR35

Sorry for my delay! I already fixed that as well :)

File: README.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Goals
 -----
 
 The key goal is to provide a base library for the
-`GitHub API <https://developer.github.com/>`_ which performs no I/O of its own (a
+`GitHub API <https://docs.github.com/en>`_ which performs no I/O of its own (a
```
Comment (brettcannon) [inline README.rst:35]:
```suggestion
`GitHub API <https://docs.github.com/>`_ which performs no I/O of its own (a
```

**Review by brettcannon — CHANGES_REQUESTED** [review]

File: README.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 gidgethub
 =========
-An asynchronous `GitHub API <https://developer.github.com/>`_ library.
+An asynchronous `GitHub API <https://docs.github.com/en>`_ library.
```
Comment (brettcannon) [inline README.rst:3]:
```suggestion
An asynchronous `GitHub API <https://docs.github.com/>`_ library.
```

**Review by brettcannon — APPROVED** [review]

**Review by Mariatta — APPROVED** [review]
Thanks!

**Pull request merged**